### PR TITLE
feat:Adding labels column to  github_issues and github_pull_requests

### PR DESCRIPTION
**Pull request by ramiroJCB**
- [x] Upgrades `mergestat/mergestat` dep to include latest version which has `labels` columns
- [x] Adds `labels` columns to postgres and starts syncing them

### CODE REVIEW

File: migrations/20220901152542_labels_github_issues_request.up.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+BEGIN;
+ALTER TABLE public.github_issues
+ADD COLUMN labels jsonb DEFAULT jsonb_build_object() NOT NULL;
```
Comment (patrickdevivo) [inline migrations/20220901152542_labels_github_issues_request.up.sql:3]:
```suggestion
ADD COLUMN labels jsonb DEFAULT jsonb_build_array() NOT NULL;
```

File: migrations/20220901152542_labels_github_issues_request.up.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+
+ALTER TABLE public.github_pull_requests
+ADD COLUMN labels jsonb DEFAULT jsonb_build_object() NOT NULL;
```
Comment (patrickdevivo) [inline migrations/20220901152542_labels_github_issues_request.up.sql:7]:
```suggestion
ADD COLUMN labels jsonb DEFAULT jsonb_build_array() NOT NULL;
```

File: migrations/20220902205214_labels_github_issues_request.up.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+BEGIN;
+ALTER TABLE public.github_issues
+ADD COLUMN labels jsonb DEFAULT jsonb_build_array() NOT NULL;
+
+
+ALTER TABLE public.github_pull_requests
+ADD COLUMN labels jsonb DEFAULT jsonb_build_array() NOT NULL;
+
+COMMIT;
```
Comment (patrickdevivo) [inline migrations/20220902205214_labels_github_issues_request.up.sql:9]:
```suggestion
COMMIT;

```

File: internal/syncer/github_repo_issues.go [hunk 4]
```diff
@@ -68,10 +69,18 @@ func (w *worker) sendBatchGitHubRepoIssues(ctx context.Context, tx pgx.Tx, repo
...
+		"labels",
 	}
 
 	inputs := make([][]interface{}, 0, len(batch))
+
 	for _, issue := range batch {
+
+		if issue.Labels == nil {
+			defaultLabel := []byte("[]")
+			issue.Labels = defaultLabel
```
Comment (patrickdevivo) [inline internal/syncer/github_repo_issues.go:81]:
```suggestion
			issue.Labels = []byte("[]")
```

File: internal/syncer/github_repo_prs.go [hunk 5]
```diff
@@ -102,10 +103,18 @@ func (w *worker) sendBatchGitHubRepoPRs(ctx context.Context, tx pgx.Tx, repo uui
...
+		"labels",
 	}
 
 	inputs := make([][]interface{}, 0, len(batch))
+
 	for _, pr := range batch {
+
+		if pr.Labels == nil {
+			defaultLabel := []byte("[]")
+			pr.Labels = defaultLabel
```
Comment (patrickdevivo) [inline internal/syncer/github_repo_prs.go:115]:
```suggestion
			pr.Labels = []byte("[]")
```

Comment (ramiroJCB) [reply internal/syncer/github_repo_prs.go]: Nice catch , unnecessary  variable

File: migrations/20220902205214_labels_github_issues_request.up.sql (hunk 3 above)
Comment (patrickdevivo) [inline migrations/20220902205214_labels_github_issues_request.up.sql:9]:
```suggestion
COMMIT;

```

**Review by patrickdevivo — APPROVED** [review]

**Pull request merged**